tapchannelmsg/wire_msgs.go

```diff
@@ -237,6 +319,8 @@ type AssetFundingCreated struct {
 	// funding output to be able to create the aux funding+commitment
 	// blobs.
 	FundingOutputs tlv.RecordT[tlv.TlvType1, AssetOutputListRecord]
+
+	// TODO(roasbeef): need to chunk this??
```

Comment:
I would say no, or not yet. We currently only allow 3 asset IDs to be committed to a channel (`maxNumAssetIDs`). And because the proofs for this output should be quite small (only one on-chain funding output and a change output), and only proof suffixes, I doubt we'd get up to 64k.